src/components/Facility/StaffCapacity.tsx

```
@@ -89,53 +86,50 @@ export const StaffCapacity = (props: DoctorCapacityProps) => {
         errors[field] = t("field_required");
-        invalidForm = true;
-      }
-      if (field === "count" && state.form[field] < 0) {
-        errors[field] = "Staff count cannot be negative";
-        invalidForm = true;
+        validForm = false;
+      } else if (field === "count" && Number(form[field]) < 0) {
+        if (Number(form[field]) < 0) {
+          errors[field] = "Staff count cannot be negative";
+          validForm = false;
+        } else if (isNaN(form[field])) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)